selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/OrganizationMembersTest.java

```diff
@@ -32,93 +31,82 @@
 import org.testng.annotations.Test;
 
 /** @author Sergey Skorik */
-public class OrganizationTest {
+public class OrganizationMembersTest {
+  private static final String PRE_CREATED_ORG_NAME = generate("organization", 5);
+  private static final String NEW_ORG_NAME = generate("organization", 5);
```

Comment:
It's better to add **"new"** prefix to the name to simplify test result analysis.